00 Core/MWSE/mods/tew/AURA/Ambient/Outdoor/outdoorMain.lua

```diff
@@ -240,13 +241,13 @@ local function resetWindoors(e)
     if table.empty(cellData.windoors)
     or not moduleInteriorWeather
     or not playInteriorAmbient
-    or not sounds.currentlyPlaying(moduleName) then
+    or not modules.getWindoorPlaying(moduleName) then
```

Comment:
What's the difference here? Asking because I just finished fixing an obscure edge case (9916a77dc83dfe31da14a0d260d5a541f9883b0c, 6760be396029df92cc1814fb1566652d2d3d6ba5, ff2175d510d9498b583f3496d86e816a9e9aafc7, 515442508440f529bcfb0516d782501e64d9f742) and I'm worried whether anything else might break. I feel we're on a very thin ice with the whole windoor/useLast logic :grin:

Reply:
No difference but since I needed a `getExteriorDoorPlaying()` for `exteriorDoors`, I thought why not make a similar one for `windoors`. Also makes what's inside `volumeController.adjustVolume()` more readable.